packages/react/src/components/Authenticator/Router/Router.tsx

```diff
@@ -22,7 +22,7 @@ const getRouteComponent = (route: string): RouteComponent => {
     case 'authenticated':
     case 'idle':
     case 'setup':
-    case 'autoSignIn':
+    case 'transition':
```

Comment:
I was chatting to @wlee221 about this. The original `autoSignIn` route was never documented, since it's an unusual route that only occurs between signing up and sign in. 

If we were to decide this route needs to be documented, then we need to make an update to Vue so it's in parity with Angular and React. This new route was renamed in the `facade.ts` file that Vue doesn't use for routes. 

We should add this documentation to a future PR where we can make the changes for Vue, so all implementations have the same name for this route.